tests/AerospikeCacheTest.php

```diff
@@ -0,0 +1,202 @@
+<?php
+declare(strict_types=1);
+
+namespace Lmc\AerospikeCache;
+
+use PHPUnit\Framework\TestCase;
+
+class AerospikeCacheTest extends TestCase
```

Comment:
I think it should extends `Lmc\Aerospike\AbstractTestCase`

Reply:
Changed